iphone/Classes/TiUIWebView.m

```diff
@@ -775,10 +775,9 @@ - (void)webViewDidFinishLoad:(UIWebView *)webView
         }
     }
     
-    // Disable user selection and the attached callout
-    BOOL disableSelection = [TiUtils boolValue:[[self proxy] valueForKey:@"disableSelection"] def:NO];
+    // Disable the context menu when selecting text
+    BOOL disableSelection = [TiUtils boolValue:[[self proxy] valueForKey:@"disableContextMenu"] def:NO];
```

Comment:
might want to rename the variable to match the property: disableContextMenu - rather than disableSelection.